libr/core/cmd.c

```diff
@@ -1504,14 +1503,19 @@ static char *parse_tmp_evals(RCore *core, const char *str) {
 		if (eq) {
 			*eq = 0;
 			const char *ov = r_config_get (core->config, kv);
-			r_strbuf_appendf (buf, "e %s=%s;", kv, ov);
+			size_t kv_len = strlen (kv);
+			size_t ov_len = strlen (ov);
+			size_t cmd_size = kv_len + ov_len + 5;
+			char *cmd = malloc (cmd_size);
```

Comment:
next time please check the result of malloc.

Reply:
No. Use r._str_newf instead of handmade malloc+snprintf